docs/quickstart.md

```diff
@@ -0,0 +1,181 @@
+# Installation Quickstart
+
+- [Setup Repository](#setup-repository)
+- [Setup Variables](#setup-variables)
+- [Setup Data Files](#setup-data-files)
+- [Start Install](#start-install)
+- [Post Install](#post-install)
+- [Cluster Access](#cluster-access)
+- [Clean up](#clean-up)
+
+
+## Setup Repository
+
+Clone this git repository on the client machine:
+```
+git clone https://github.com/ocp-power-automation/ocp4-upi-powervm.git
+cd ocp4_upi_powervm
+```
+
+## Setup Variables.
+
+Update the var.tfvars with values explained in the following sections. You can also set the variables using other ways mentioned [here](https://www.terraform.io/docs/configuration/variables.html#assigning-values-to-root-module-variables) such as -var option or environment variables.
+
+### Setup PowerVC Environment Variables
```

Comment:
This way of organizing the variables makes it much easier to understand. Thanks.
Is it possible to group the `Required` and `Optional` together ?

Reply:
Do we want a new sub-section on top under variables with only all the required variables?

Reply:
Or we just keep a generic note that you need to fill in all the variables marked as (Required) below?

Reply:
no subsection.. just changing the order.
From this
```
var1 (required)
var2 (required)
var3 (optional)
var4 (required)
```
to the following
```
var1 (required)
var2 (required)
var4 (required)
var3 (optional)
```